test/unit/accept_language_test.rb

```diff
@@ -4,15 +4,23 @@
 class AcceptLanguageTest < Minitest::Test
   def assert_language(item, expect = {})
     assert_equal expect[:code], item.code, "failed code comparison"
-    assert_equal expect[:region], item.region, "failed region comparison"
+
+    if expect[:region].nil?
```

Comment:
Why is this being added if you're not calling it anywhere on tests you wrote in this PR?